### DISCUSSION TITLE
Application error: a client-side exception has occurred

### DISCUSSION THREAD

**Discussion by oajko** [supabase/supabase#17672] [Questions]
The 1st image below keeps popping up only when i go onto supabase wrappers page. nothing else.

It first occurred when I was trying to implement Stripe foreign wrappers. I was following the documentation, when then occurred when I created a server for stripe_server (following documentation).

The bottom image shows the code part of documentation which caused this application error.

Any ideas on how to fix this issue?

![2023-09-23 17_37_55-Application error_ a client-side exception has occurred](https://github.com/supabase/supabase/assets/102440460/dbaa303a-d7cd-415c-af43-650a75c80bc3)
![2023-09-23 17_37_16-Stripe _ Supabase Docs](https://github.com/supabase/supabase/assets/102440460/b5870f73-676a-488a-b4d7-35793fa3fdd8)

**Comment by NeoPrint3D** [comment]
Experiencing the same issue
